### PR TITLE
dev-java/tomcat-native: add 2.0.2 

### PR DESCRIPTION
**Pull request by vaukai**

### CODE REVIEW

**Comment by fordfrog** [comment]
i still don't have openssl-3 on my systems, so i cannot test it. does it really compile without issues using jdk 1.8?

**Comment by vaukai** [comment]
> i still don't have openssl-3 on my systems, so i cannot test it. does it really compile without issues using jdk 1.8?

Just re-tested: [build.log.txt](https://github.com/gentoo/gentoo/files/10302495/build.log.txt). It does.

File: dev-java/tomcat-native/tomcat-native-2.0.2.ebuild [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+	dev-libs/openssl:0/3
+	>=virtual/jre-1.8:*"
+
+DEPEND=">=virtual/jdk-1.8:*
+	test? ( dev-java/ant-junit:0 )"
+
+BDEPEND="verify-sig? ( sec-keys/openpgp-keys-apache-tomcat-connectors )"
+VERIFY_SIG_OPENPGP_KEY_PATH="${BROOT}/usr/share/openpgp-keys/tomcat-connectors.apache.org.asc"
+
+S=${WORKDIR}/${P}-src
```
Comment (ConiKost) [inline dev-java/tomcat-native/tomcat-native-2.0.2.ebuild:29]: After SRC_URI please. See skell.ebuild.

**Comment by fordfrog** [comment]
> > i still don't have openssl-3 on my systems, so i cannot test it. does it really compile without issues using jdk 1.8?
> 
> Just re-tested: [build.log.txt](https://github.com/gentoo/gentoo/files/10302495/build.log.txt). It does.

cool, thanks! i was searching upstream site and i was not able to find any mention about restriction to java 11+ anymore.

File: dev-java/tomcat-native/tomcat-native-2.0.2.ebuild (hunk 1 above)
Comment (fordfrog) [reply dev-java/tomcat-native/tomcat-native-2.0.2.ebuild:29]: i adjusted that during merge

**Pull request closed without merging**